gtd-timer/Controllers/SignUpController.cs

```diff
@@ -22,6 +22,11 @@ public SignUpController(IUnitOfWork uow)
             this.unitOfWork = uow;
         }
 
+        /// <summary>
+        /// Retrieve user by his/her Id.
+        /// </summary>
+        /// <param name="id">The Id of the desired User</param>
+        /// <returns>A string status</returns>
```

Comment:
misleading - it doesn't return string. It returns object that is later transformed into HTTP response. Let's leave returns tag empty here